### DISCUSSION TITLE
хуета

### DISCUSSION THREAD

**Discussion by saberkovich** [stoatchat/discussions#786] [Feedback]
del

**Comment by Mopo3eX** [comment]
Да норм, 2 часа на регистрацию, час создания сервера и час загрузки одного чата, всё норм )

**Reply by Archmaner** [reply]
хз у меня все ворк

**Reply by amycatgirl** [reply]
Revolt is under heavy load currently, sorry for the inconvenience!

**Reply by ulcuber** [reply]
> Revolt is under heavy load currently, sorry for the inconvenience!

It's probably because discord was blocked in our country and many people need alternative
